dhis-2/dhis-support/dhis-support-artemis/src/main/java/org/hisp/dhis/artemis/audit/listener/PostDeleteAuditListener.java

```diff
@@ -67,15 +68,24 @@ public void onPostDelete( PostDeleteEvent postDeleteEvent )
     {
         Object entity = postDeleteEvent.getEntity();
 
-        getAuditingScope( entity ).ifPresent( scope -> {
+        getAuditable( entity ).ifPresent( auditable -> {
+            boolean shouldAudit = Arrays.stream( auditable.eventType() )
```

Comment:
I'd refactor this method into the superclass, so that we can do `boolean isAuditable = getAuditable(entity, "delete");